test/pid_controller_test.rb

```diff
@@ -105,9 +105,9 @@ def test_update_flow
     @controller.update
     # control signal = Ki * integral + Kp * p_value + Kd * (p_value - previous_p_value) / dt
     # p_value = (error_rate - ideal_error_rate) - rejection_rate = (0.11 - 0.01) - 0 = 0.1
-    # = 0.1 * (-0.01+ 0.1) * 10 + 1 * 0.1 + 0.01 * (0.1 - (-0.01)) / 10 = 0.19011
-    # rejection rate = 0 + 0.19011 = 0.19011
-    assert_in_delta(@controller.metrics[:rejection_rate], 0.19, 0.01)
+    # With anti-windup, integral behavior changes when output is saturated
```

Comment:
could you add a comment with what the new calculation (including the integral anti-windup is?) That would help people looking at the test in the future